aim/tools/cli/commands/infra.py

```diff
@@ -56,3 +56,32 @@ def create(ctx):
     ctx.obj['apic_manager'].ensure_infra_created_on_apic()
     ctx.obj['apic_manager'].ensure_bgp_pod_policy_created_on_apic()
     ctx.obj['apic_manager'].ensure_opflex_client_cert_validation_disabled()
+
+
+@infra.command(name='tag-list')
+@click.option('--tag-name', '-t', help='Tag name')
+@click.pass_context
+def tag_list(ctx, tag_name):
+    ctx.obj['apic_manager'].ensure_infra_created_on_apic()
```

Comment:
Why do we have to call ensure_infra_created_on_apic() here?

Reply:
Oops. We don't. Will remove.